pytorch_lightning/__init__.py

```diff
@@ -76,6 +67,33 @@
         'seed_everything',
         'metrics',
     ]
+else:
+    try:
+        # This variable is injected in the __builtins__ by the build
+        # process. It used to enable importing subpackages of skimage when
+        # the binaries are not built
+        _ = None if __LIGHTNING_SETUP__ else None
+    except NameError:
+        __LIGHTNING_SETUP__: bool = False
```

Comment:
does this has to be in the else branch?

Reply:
It does not - moved it out ! Good catch :)